### PR TITLE
New package: CloudFoundry.CLI.v7 version 7.7.7

### PR DESCRIPTION
**Pull request by AshhubAli**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137254)

### CODE REVIEW

**Comment by NightWalker7558** [comment]
The Installer URLs mentioned in the manifest are taken from the [official GitHub repository](https://github.com/cloudfoundry/cli/releases/tag/v7.7.7) of Cloud Foundry CLI that also link to the Amazon S3 Stores.

File: manifests/c/CloudFoundry/CLI/v7/7.7.7/CloudFoundry.CLI.v7.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+PackageIdentifier: CloudFoundry.CLI.v7
+PackageVersion: 7.7.7
+PackageLocale: en-US
+Publisher: Cloud Foundry Foundation
+PublisherUrl: https://cli.cloudfoundry.org/
+PublisherSupportUrl: https://github.com/cloudfoundry/cli/issues
+# PrivacyUrl: 
+Author: Cristhian Peña
+PackageName: Cloud Foundry CLI v7
+PackageUrl: https://cli.cloudfoundry.org/en-US/v7/
```
Comment (stephengillie) [inline manifests/c/CloudFoundry/CLI/v7/7.7.7/CloudFoundry.CLI.v7.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://github.com/cloudfoundry/cli/releases
```

**Comment by stephengillie** [comment]
> The Installer URLs mentioned in the manifest are taken from the [official GitHub repository](https://github.com/cloudfoundry/cli/releases/tag/v7.7.7) of Cloud Foundry CLI that also link to the Amazon S3 Stores.

Yes - the requirement here is that anyone (such as myself) can navigate from the PackageUrl to each InstallerUrl.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**